core/src/test/scala/org/apache/spark/StatusTrackerSuite.scala

```diff
@@ -61,7 +61,7 @@ class StatusTrackerSuite extends SparkFunSuite with Matchers with LocalSparkCont
     // Passing `null` should return jobs that were not run in a job group:
     val defaultJobGroupFuture = sc.parallelize(1 to 1000).countAsync()
     val defaultJobGroupJobId = eventually(timeout(10.seconds)) {
-      defaultJobGroupFuture.jobIds.head
+      defaultJobGroupFuture.jobIds.max
     }
```

Comment:
Revert ? There should only be a single job submitted here, so `head` will be `max`